free-3.md

```diff
@@ -9,41 +9,100 @@ good way to meet their needs.
+concrete plan of action that they can take. It helps people who struggle with 
+productivity but might be overwhelmed on how to overcome it. There are many
+productivity apps and task managers, but this language would allow someone
+to input their tasks and randomly schedule the tasks into a daily calendar
+based on the length, priority, and frequency of the task. Having this 
+definite scheduling puts more pressure on the user to complete the tasks and
+allow them to spend their time more efficiently.
 
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
+A DSL would allow the user to specify the nature of the tasks they need to
+accomplish and be able to type out these tasks more efficiently. A GUI might be
+easier to navigate initially, but having the user to input tasks through code
+would allow them to be more specific with the conditions and parameters associated
+with each task. The computer can then randomly generate an hourly schedule filled 
+with the tasks and gives the user a visual output that outlines the schedule.
 
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
+The idea excites me because I know that nearly all of my friends in college have
+struggled with procrastination. I don't think people need to or should be productive
+all the time, but when you really need motivation to be productive, it's nice to
+have a really clear plan of action. Coming up with this action plan is difficult
+and time-consuming to do on your own, so having a language to express the tasks you
+need to accomplish and having a schedule computed for you could be an effective tool.
+I came up with this idea after realizing how much time I spend adding things to my
+task manager and then scheduling tasks in my calendar which sort of negates the
+sort of productivity I'm trying to achieve.
 
 
 ### Domain
 _Describe the project's domain in five words._
+Task organization, automatic schedule generation
 
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
+The user would input their tasks in a simple text file. They would create
+task objects that could have several parameters including predicted length,
+priority, deadline, and frequency. The user could then group the tasks together
+into different categories of tasks, such as homework, jobs, self-care, etc. The
+idea is to have the language feel like the user is sort of writing down notes
+about the things they need to do, and leave the organization and scheduling to
+the computer.
 
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
+The computer will associate a certain number to each task, and this number 
```

Comment:
"Do you know anything about this domain? Do you know of existing DSLs that they didn't mention or do you have some expertise thinking about or writing programs for this domain? How might your experience augment your partner's thoughts on the domain?" Again, I did a similar project for a Hackathon, and it sounds like we have similar ideas regarding calculating priority. 

Will you ask the user question to help him/her gauge the priority of a task, or are you asking him/her to determine that for him/herself?

It might be interesting to incorporate the order of the tasks / optimal time to do each task according to the user's existing commitments.